lib/transaction_prepare.c

```diff
@@ -100,10 +100,12 @@ compute_transaction_stats(struct xbps_handle *xhp)
 
 		if ((ttype != XBPS_TRANS_CONFIGURE) && (ttype != XBPS_TRANS_REMOVE) &&
```

Comment:
wouldn't it be easier to add that here too?

Reply:
Right.
I incorrectly assumed, that `xbps_dictionary_set_bool(obj, "download", true);` must not be ommited here for `--download-only`